README.md

```diff
@@ -4,8 +4,20 @@
 
 # Jaeger - a Distributed Tracing System
 
-  * See [documentation](http://jaeger.readthedocs.io/en/latest/) for getting started, operational details, and other information.
-  * Read a blog post [Evolving Distributed Tracing at Uber](https://eng.uber.com/distributed-tracing/).
+Jaeger, inspired by [Dapper][dapper] and [OpenZipkin](http://zipkin.io),
+is a distributed tracing system released as open source by [Uber Technologies][ubeross].
+It can be used for monitoring microservice-based architectures:
+
+  * Distributed context propagation
+  * Distributed transaction monitoring
+  * Root cause analysis
+  * Service dependency analysis
```

Comment:
We should also document somewhere that the dependencies tab relies on a cron job that we haven't open sourced yet. I believe that users get a blank page when they hit the dependencies tab.